vtiger_connector_base/__manifest__.py

```diff
@@ -1,20 +1,17 @@
 # -*- coding: utf-8 -*-
+# See LICENSE file for full copyright and licensing details.
+
 {
     'name': 'VTiger Base Connector',
-    'version': '10.0.1.0.0',
-    'summary': 'VTiger Base Connector',
-    'description': """
-        VTiger Base Connector""",
+    'version': '11.0.1.0.0',
+    'category': 'Hidden',
```

Comment:
kem Hidden ? Should not it be extra tools?
@saif-serpentcs kem aam dost?  logic nai lagavvanu?

Reply:
@JayVora-SerpentCS can i add **Generic Module** or **Base Module**?